### ISSUE TITLE
Multiple input columns to UDTFs

### ISSUE THREAD

**Issue by pearu** [xnd-project/rbc#148]
Original issue: #124 

Wish:
```
@omnisci('int32(Column<double>, Column<double>, int64|sizer=RowMultiplier, Column<double>)')
def add_columns(x, y, m, r):
    input_row_count = len(x)
    for i in range(input_row_count):
        for c in range(m):
            j = i + c * input_row_count
            r[j] = x[i] + y[i]
    return m * input_row_count
 # SQL query: select f8 from table(add_columns(cursor(select f8 from {omnisci.table_name}), cursor(select f8 from mytable), 1));
```

Failure:
```
RelAlgDagBuilder.cpp:2303 Check failed: size_t(1) == inputs.size() (1 == 2)
```

Fix:
- [x] https://github.com/omnisci/omniscidb-internal/pull/4696

**Comment by pearu** [comment]
The upstream PR is merged, enable the corresponding rbc tests before closing this issue.

**Comment by pearu** [comment]
Closed via #153
